### ISSUE TITLE
Missing holidays in Spain

### ISSUE THREAD

**Issue by gdordal** [dr-prodigy/python-holidays#931]
Hi,

Holiday Easter Monday (10th of April) is missing in the CT subdivision in Spain.

Thanks,

**Comment by KJhellico** [comment]
Seems to be fixed in #902. Please, check with beta version, if you can.

**Comment by gdordal** [comment]
Looks good in v0.20 
When will be released?

**Comment by arkid15r** [comment]
> Looks good in v0.20 When will be released?

It's my plan to release it within the next 2 weeks (pessimistic ETA -- by end of this month)

**Comment by gdordal** [comment]
ok, thanks a lot!

**Comment by cdeharo** [comment]
Hi, is this fix included in 0.21.13? Today is being shown as a working day to me.

**Comment by KJhellico** [comment]
Hi, @cdeharo! Yes, it was included in 0.20. I've just checked:
```
>>> import holidays
>>> ct = holidays.ES(subdiv="CT")
>>> ct["2023-04-10"]
'Lunes de Pascua'
>>> holidays.__version__
'0.20'
```
Same with 0.21.13.

**Comment by arkid15r** [comment]
> Hi, is this fix included in 0.21.13? Today is being shown as a working day to me.

HI @cdeharo,
the issue was resolved in v0.20 and closed by the OP.

Please open a new ticket if you're still experiencing holidays inconsistency.
